notebooks/example/anomaly_code.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    len(records)
</pre><p>Hmm inspecting the csv file on jupyterlab shows 576 rows, but here I see 574 records, any idea what might cause this inconsistency?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/HTM-applications/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>